src/frameworks/drupal7/drush.md

```diff
@@ -10,11 +10,41 @@ commands at [Drush.org](http://www.drush.org). If you have never used
 Drush before, you can learn more about it on the [Drush GitHub
 Repository](https://github.com/drush-ops/drush#description)
 
-Platform.sh has Drush installed by default, so all environments can utilize Drush commands in the development process. You can use the CLI to set up Drush aliases, to easily run Drush commands on
-specific remote Platform.sh environments.
+- Platform.sh's Drupal templates have Drush installed automatically.
 
-> **note**
-> Platform's CLI requires **Drush 6 or greater**.
+- All custom environments can utilize Drush commands in the development process if they have `drush` configured as a composer requirement and added it to the environment's `$PATH`. See the section below on [install Drush in custom projects](#install-drush-in-custom-projects)
+
+- In addition, you can use the [Platform.sh CLI](/overview/cli.md) to set up Drush aliases easily for all of your project's environments. See the section below on [use drush aliases](#use-drush-aliases)
+
+## Installing Drush
+
+## Install Drush on Platform.sh-provided Drupal examples
+
+Platform.sh's Drupal templates have Drush installed automatically via Composer and should not require you to add it.
+
+## Install Drush in custom projects
+
+### Make Drush a project requirement
+
+Run this command in the project's repository root folder:
+```bash
+$ composer require drush/drush
+```
+Then, commit and push.
+
+Drush will then be available at `vendor/bin/drush`, in the exact same version on your local system and on Platform.sh.
+
+### Add Drush to the project's $PATH
+
+Add a new file named `.environment` to the root of your your project's git repository with this code:
+```bash
+# Statements in this file will be executed (sourced) by the shell in SSH
+# sessions, in deploy hooks, in cron jobs, and in the application's runtime
+# environment.
+
+# Allow executable app dependencies from Composer to be run from the path.
+export PATH=/app/vendor/bin:$PATH
```

Comment:
Let's use the same code from the example that lets the use of the same `.environment` file in PE.

Reply:
Concur.